driver-core/src/test/functional/com/mongodb/operation/CreateCollectionOperationSpecification.groovy

```diff
@@ -185,6 +185,29 @@ class CreateCollectionOperationSpecification extends OperationFunctionalSpecific
         }
     }
 
+    def 'should create collection in respect to the autoIndex option'() {
+        given:
+        assert !collectionNameExists(getCollectionName())
+
+        when:
+        new CreateCollectionOperation(getDatabaseName(), getCollectionName())
+                .autoIndex(autoIndex)
+                .execute(getBinding())
+
+        then:
+        def stats = new CommandWriteOperation<Document>(getDatabaseName(),
+                new BsonDocument('collStats', new BsonString(getCollectionName())),
+                new DocumentCodec()).execute(getBinding())
+
+        expect:
```

Comment:
Closer... Generally there is no need for to follow "then:" with "expect:".  Just get rid of the "expect:" and even the stats local variable.